spandx.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  host: "localhost",
```

Comment:
It'd be nice if there was a .env file or something for those of us with weird setups 😛